src/transformers/modeling_tf_lxmert.py

```diff
@@ -0,0 +1,1248 @@
+            Prediction scores of the textual matching objective (classification) head (scores of True/False
+            continuation before SoftMax).
+        question_answering_score: (:obj:`tf.Tensor` of shape :obj:`(batch_size, n_qa_answers)`):
+            Prediction scores of question answering objective (classification).
+        language_hidden_states (:obj:`tuple(tf.Tensor)`, `optional`, returned when ``output_hidden_states=True`` is passed or when ``config.output_hidden_states=True``):
+            Tuple of :obj:`tf.Tensor` (one for input features + one for the output of each cross-modality layer)
+            of shape :obj:`(batch_size, sequence_length, hidden_size)`.
+        vision_hidden_states (:obj:`tuple(tf.Tensor)`, `optional`, returned when ``output_hidden_states=True`` is passed or when ``config.output_hidden_states=True``):
+            Tuple of :obj:`tf.Tensor` (one for input features + one for the output of each cross-modality layer)
+            of shape :obj:`(batch_size, sequence_length, hidden_size)`.
+        language_attentions (:obj:`tuple(tf.Tensor)`, `optional`, returned when ``output_attentions=True`` is passed or when ``config.output_attentions=True``):
+            Tuple of :obj:`tf.Tensor` (one for each layer) of shape
+            :obj:`(batch_size, num_heads, sequence_length, sequence_length)`.
+            Attentions weights after the attention softmax, used to compute the weighted average in the self-attention
+            heads.
+        vision_attentions (:obj:`tuple(tf.Tensor)`, `optional`, returned when ``output_attentions=True`` is passed or when ``config.output_attentions=True``):
+            Tuple of :obj:`tf.Tensor` (one for each layer) of shape
+            :obj:`(batch_size, num_heads, sequence_length, sequence_length)`.
+            Attentions weights after the attention softmax, used to compute the weighted average in the self-attention
+            heads.
+        cross_encoder_attentions (:obj:`tuple(tf.Tensor)`, `optional`, returned when ``output_attentions=True`` is passed or when ``config.output_attentions=True``):
+            Tuple of :obj:`tf.Tensor` (one for each layer) of shape
+            :obj:`(batch_size, num_heads, sequence_length, sequence_length)`.
+            Attentions weights after the attention softmax, used to compute the weighted average in the self-attention
+            heads.
+
+    """
+
+    loss: [tf.Tensor] = None
+    prediction_logits: Optional[tf.Tensor] = None
+    cross_relationship_score: Optional[tf.Tensor] = None
+    question_answering_score: Optional[tf.Tensor] = None
+    language_hidden_states: Optional[Tuple[tf.Tensor]] = None
+    vision_hidden_states: Optional[Tuple[tf.Tensor]] = None
+    language_attentions: Optional[Tuple[tf.Tensor]] = None
+    vision_attentions: Optional[Tuple[tf.Tensor]] = None
+    cross_encoder_attentions: Optional[Tuple[tf.Tensor]] = None
+
+
+class TFLxmertVisualFeatEncoder(tf.keras.layers.Layer):
```

Comment:
nit: I think `TFLxmertVisualFeatureEncoder` would be more consistent with our usual naming patterns.